Maya/Exporter/BabylonExporter.Mesh.cs

```diff
@@ -939,6 +963,102 @@ private GlobalVertex ExtractVertex(MFnMesh mFnMesh, int polygonId, int vertexInd
                 }
             }
 
+            indexUVSet = 2;
+            if (uvSetNames.Count > indexUVSet && isUVExportSuccess[indexUVSet])
+            {
+                try
+                {
+                    float u = 0, v = 0;
+                    mFnMesh.getPolygonUV(polygonId, vertexIndexLocal, ref u, ref v, uvSetNames[indexUVSet]);
+                    vertex.UV3 = new float[] { u, v };
+                }
+                catch
+                {
+                    // An exception is raised when a vertex isn't mapped to an UV coordinate
+                    isUVExportSuccess[indexUVSet] = false;
+                }
+            }
```

Comment:
Use a helper function?

Reply:
Generally, all of these repeat codes can use some helpful functions. Not a big deal, but it's a lot of duplication.